css/about.css

```diff
@@ -0,0 +1,152 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
When splitting out CSS files you can also create a 3rd CSS file and put your normalize/reset CSS in that and just reference it before index.css and about.css on each page, that way you don't have to keep copying it into each CSS page.